ocs_ci/ocs/resources/csv.py

```diff
@@ -0,0 +1,66 @@
+"""
+CSV related functionalities
+"""
+import logging
+
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.exceptions import CommandFailed
+from ocs_ci.utility.utils import TimeoutSampler
+
+
+logger = logging.getLogger(__name__)
+
+
+class CSV(OCP):
+    def __init__(self, name="", *args, **kwargs):
+        """
+        Initializer function for CSV class
+
+        Args:
+            name (str): Name of CSV
+
+        """
+        super(CSV, self).__init__(*args, **kwargs)
+        self.name = name
+
+    def check_phase(self, phase):
+        """
+        Check phase of CSV resource
+
+        Args:
+            phase (str): Phase of CSV object
+
+        Returns:
+            bool: True if phase of object is the same as passed one, False
+                otherwise.
+
+        """
+        data = {}
+        try:
+            data = self.get(resource_name=self.name)
+        except CommandFailed:
+            logger.info(f"Cannot find CSV object {self.name}")
+        try:
+            return data['status']['phase'] == phase
```

Comment:
use .get()

Reply:
I am expecting KeyError if the key is not defined here, don't want to use get here

Reply:
I think that in Raises section we should document just exception which are really raised in the code itself. Or the exception which you think your code can fail with and you don't handle this exception than user who is calling that function can except this exception will be raised by calling this method and user should handle this exception by himself.  As I am excepting this Exception KeyError and handling this error in the code and not re raise it I think no reason to document it in Raises section.